File: factorial/factorial.py
num = int(input("input any number: "))

def calcFactorial(num):
    factorial = 1

    for i in range(1, num+1):
        factorial *= i
    
    return factorial 

def showFactorial(num):
    factorial = calcFactorial(num)
    return print(factorial)

showFactorial(num)

File: factorial/factorial.js
let calcFactorial = (num) => {
    let factorial = 1;
    for (let i = 1; i < num + 1; i++){
        factorial *= i
    }
    return factorial;
}
 let showFactorial = (num) => {
    factorial  = calcFactorial(num);
    return console.log(factorial)
 }

showFactorial(4)
